filebeat/docs/inputs/input-kafka.asciidoc

```diff
@@ -0,0 +1,107 @@
+:type: kafka
+
+[id="{beatname_lc}-input-{type}"]
+=== Kafka input
+
+++++
+<titleabbrev>Kafka</titleabbrev>
+++++
+
+Use the `kafka` input to read from topics in a Kafka cluster.
+
+To configure this input, specify a list of <<hosts,`hosts`>> to use for this
+cluster, a list of <<topics,`topics`>> to track, and a <<groupid,`group_id`>>
+to connect with.
```

Comment:
We should note that the brokers mentioned are only used for bootstrapping the actual connections. One doesn't need to enter all hosts of a cluster here.